keystore/kms/aws/client.go

```diff
@@ -106,12 +106,21 @@ func (e *KMSClient) ListAliases(ctx context.Context, keyID *string) ([]types.Ali
 		input.KeyId = keyID
 	}
 
-	result, err := e.client.ListAliases(ctx, input)
-	if err != nil {
-		return nil, err
+	aliases := make([]types.AliasListEntry, 0)
```

Comment:
as I understand, here we expect at least 1 entry, am i correct? if so, then lets initialize with size of 1? or if it list a lot of aliases, then lets initialize with meaningful amount of entries. for example if here is only client names then lets use number 8 (we use 2 for clientID in tests and 2 for zoneID).

Reply:
Yes, it's a good question. The default AWS pagination size is 50 items and we can specify how many items we want to return per page. For example, during the key making, we will call this function on every key, even if no aliases have been specified yet, so we will do some redundant allocation.